src/display/annotation_layer.js

```diff
@@ -725,6 +725,17 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
               // tooltip
               event.target.title = detail.userName;
             },
+            display() {
```

Comment:
It looks like display and print are repeated, can they live on WidgetAnnoationElement or are they not valid for all annotation types? Alternatively, would it make sense to add another class for form widgets to extend from?